src/sensor_imu_plugin.cpp

```diff
@@ -56,6 +54,40 @@ SensorImuPlugin::SensorImuPlugin(JSBSim::FGFDMExec* jsbsim)
 
 SensorImuPlugin::~SensorImuPlugin() {}
 
+void SensorImuPlugin::setSensorConfigs(TiXmlElement* configs) {
```

Comment:
Better use const references:
```suggestion
void SensorImuPlugin::setSensorConfigs(const TiXmlElement &configs) {
```

Reply:
Using pointers allow us to handle the case when the element doesn't exist: (`TiXmlElement*` = nullptr ) while having const reference requires to handle before this is referenced. I have updated to const references, but was wondering using `TiXmlElement` is already not the correct approach